src/Microsoft.TemplateEngine.TemplateLocalizer.Core/TraversalRules/RegexFilteredTraversalRule.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class RegexFilteredTraversalRule : TraversalRule
         /// Creates an instance of <see cref="RegexFilteredTraversalRule"/>.
         /// </summary>
         /// <param name="regexPattern">Regex pattern string that will be used to match json element names.</param>
-        /// <param name="keyCreator"><see cref="IJsonKeyCreator"/> to be used when creating a key for the match elements.</param>
+        /// <param name="keyExtractor"><see cref="IJsonKeyCreator"/> to be used when creating a key for the match elements.</param>
```

Comment:
Ouch. We should probably rename the parameter here. Because the type is `IJsonKeyCreator`, not `IJsonKeyExtractor`.
I renamed the type previously, but forgot to rename the param.
Same goes for the case below.

Reply:
changed back to `keyCreator`